ddtrace/_trace/trace_handlers.py

```diff
@@ -666,6 +733,7 @@ def listen():
         "botocore.patched_kinesis_api_call",
         "botocore.patched_sqs_api_call",
         "botocore.patched_stepfunctions_api_call",
+        "botocore.patched_bedrock_api_call",
```

Comment:
General Q... So our ideal end state for all integrations is for each one to be registered here instead of directly accessing the `datadog_integration` product? And then all of the span management would happen in the core as well?

Reply:
The ideal end state I have in mind with this change is that given a module in contrib, it contains only instrumentation and event dispatches. The tracing code that reacts to those events could live in `ddtrace/_trace`. Does that answer your question?